packages/core/src/Stores/ui-store.js

```diff
@@ -112,7 +112,7 @@ export default class UIStore extends BaseStore {
     @observable current_focus = null;
 
     // Enabling EU users
-    @observable is_eu_enabled = false; // TODO: [deriv-eu] - Remove this constant when all EU sections are done.
+    @observable is_eu_enabled = true; // TODO: [deriv-eu] - Remove this constant when all EU sections are done.
```

Comment:
This should be remain false until we are ready to ship.

Reply:
Yes. I didn't change this file. Same for `app-modals`. Will push a fix.